eng/scripts/Deploy-TestResources.ps1

```diff
@@ -1,6 +1,8 @@
 param(
-    [Parameter(Mandatory=$true)]
-    [string]$Area,
+    [Parameter(Mandatory=$true, ParameterSetName="ByPath")]
+    [string]$Path,
+    [Parameter(Mandatory=$true, ParameterSetName="ByTool")]
+    [string]$Tool,
```

Comment:
When you have these two set as mandatory, would PS expect you to just set one of the two or both?

Reply:
One or the other, but not both

Reply:
You have to land on a parameter set, and there's a required parameter in each set